submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+                <img
+                  src="images/mastercard.svg"
+                  alt="mastercards_icon"
+                  class="icon_paymethod"
+                />
+                <img
+                  src="images/visa.svg"
+                  alt="visa_icon"
+                  class="icon_paymethod"
+                />
+              </label>
+              <input
+                type="radio"
+                name="payment"
+                id="paypal"
+                class="visually__hidden radio_input_payment"
+              />
+              <label
+                for="paypal"
+                class="form-choose_payment_radio paypal_label"
+              >
+                <img src="images/paypal.svg" alt="" class="icon_paypal" />
+              </label>
+              <input
+                type="radio"
+                name="payment"
+                id="amazon"
+                class="visually__hidden radio_input_payment"
+              />
+              <label
+                for="amazon"
+                class="form-choose_payment_radio amazon_label"
+              >
+                <img
+                  src="images/amazon.svg"
+                  alt="amazon icon"
+                  class="icon_amazon"
+                />
+              </label>
+            </div>
+            <div class="wrapper_form-card_number">
+              <input
+                type="text"
+                class="form-sign_pay_method_inputs  input_card_number validate_on_focus"
+                placeholder="0000 0000 0000 0000"
+                minlength="16"
+                maxlength="16"
+              />
+              <span class="input_card_data_title card_number_title"></span>
+              <input
+                type="datetime"
+                class=" form-sign_pay_method_inputs  input_date validate_on_focus"
+                placeholder="DD/MM/YY"
+                maxlength="6"
+                minlength="6"
+              />
+              <span class="input_card_data_title input_date_title"></span>
+              <input
+                type="text"
+                class="form-sign_pay_method_inputs  input_code validate_on_focus"
+                inputmode="numeric"
+                placeholder="000"
+                minlength="3"
+                maxlength="3"
+              />
+              <span class="input_card_data_title input_card_code_title"></span>
+            </div>
+          </section>
+        </article>
+        <section class="form-submit">
+          <div class="wrapper_agree_checkbox">
+            <input
+              type="checkbox"
+              name=""
+              id="agree"
+              class="visually__hidden"
+              required
+            />
+            <span class="custom_checkbox"></span>
+            <label for="agree" class="form-label_agree">
+              I agree to the
+              <a href="#" class="submit_link">
```

Comment:
When I clicked on link checkbox will be checked. Do we need this behavior?

Reply:
Nope, done.